packer/install-dependencies.sh

```diff
@@ -317,8 +317,8 @@ for AZURE_NPM_VERSION in ${AZURE_NPM_VERSIONS}; do
 done
 
 AZURE_VNET_TELEMETRY_VERSIONS="
+1.0.30
 1.0.29
-1.0.28
```

Comment:
ditto

Reply:
Actually, this is for the Azure NPM spec, so let's remove all changes to this array and restrict them to the Azure NPM PR

Reply:
reverted npm changes